### PR TITLE
fix(frontend)  support utf-8 charset

### PR DESCRIPTION
**Pull request by lutongzero**
It seems that StringEntity should be changed to UTF-8 encoding to support non-English full name
#6960 

## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Review by chriscollins3456 — COMMENTED** [review]
This fix makes sense to me! thanks for the contribution @lutongzero
 i have one question about a change you made though.

Can you explain the issue that you're specifically trying to fix here, though? As in what's not working?

File: datahub-frontend/app/client/AuthServiceClient.java [hunk 1]
```diff
@@ -288,7 +288,7 @@ public void track(@Nonnull final String event) {
               TRACK_ENDPOINT));
 
       // Build JSON request to track event.
-      request.setEntity(new StringEntity(event));
+      request.setEntity(new StringEntity(json, StandardCharsets.UTF_8));
```
[the remark this answers is not in the document]
Comment (jjoyce0510) [reply datahub-frontend/app/client/AuthServiceClient.java:291]: believe this should be event. I'll just change and re-run

**Review by chriscollins3456 — APPROVED** [review]

**Comment by lutongzero** [comment]
> This fix makes sense to me! thanks for the contribution @lutongzero i have one question about a change you made though.
> 
> Can you explain the issue that you're specifically trying to fix here, though? As in what's not working?

When signup a new user, if the fullname is not in the ISO-8859-1 character set, such as in non-Latin languages like Chinese or Japanese, it may cause garbled characters to appear.

**Comment by chriscollins3456** [comment]
> When signup a new user, if the fullname is not in the ISO-8859-1 character set, such as in non-Latin languages like Chinese or Japanese, it may cause garbled characters to appear.

okay thanks! i thought that was the case but just wanted to be clear we were on the same page.

**Pull request merged**